### DISCUSSION TITLE
Suppress bundle upgrades

### DISCUSSION THREAD

**Discussion by joeloff** [wixtoolset/issues#8579] [Questions]
Does Burn or wixstdba currently support conditionally suppressing bundle upgrades (the removal of the previous bundles) at install time and deferring the removal until after a reboot?

If not, should I file a feature request for v5? 

If we provided a PR, would this be considered for 3 at all? I'm 99.9% sure 3.1 is only taking security fixes, but I was asked to ask (and yes, I pointed others to the announcement).

**Comment by barnson** [comment]
Burn does. WixStdBA does not.

It sounds obscure so if you file a feature request, come up with a good user story.

As for v3...yeah, no.

**Comment by joeloff** [comment]
@barnson this is going to be somewhat crosscutting between the discussions and development areas. Are there any instructions on how to configure a build environment for wix3? Let me know if you want me to close this issue and reopen the question on the Development side.

**Comment by joeloff** [comment]
I got this sorted out and able to build WiX3. In case anyone runs into it

- Build from a 32-bit VS developer prompt, like 15.9 which uses 32-bit msbuild
- Fix up the Win SDK checks in WixBuild.props if you're on a newer version with newer SDKs. In my case, I had 10.0.7763.0 installed, but the build checked for 17134 and 14393.

**Reply by barnson** [reply]
That's a partial environment. An official build is _much_ harder, with multiple VS and SDK versions required.
